app/src/main/AndroidManifest.xml

```diff
@@ -111,6 +111,15 @@
                 android:name="android.appwidget.provider"
                 android:resource="@xml/small_widget" />
         </receiver>
+        <receiver android:name=".Very_small_widget">
```

Comment:
- It should be placed inside the widget sub-package. I can see that it is in the root package. Please make that change
- Please follow the naming convention of other receivers. Other receiver names are following PascalCasing.
- I don't see Very_small_widget and very_small_widget_info.xml files. How is this even working then? Are you sure you had committed those 2 files?